templates/expo-template-default/gitignore

```diff
@@ -9,6 +9,10 @@ dist/
 web-build/
 expo-env.d.ts
 
+# prebuild generates the native directories, so we don't want to version them
+android/
+ios/
```

Comment:
This is a major change. It will indicate to users that writing and managing their own native code is not recommended, which we've leaned away from. Could try it though.
```suggestion
/android
/ios
```